vrecord

```diff
@@ -232,6 +232,16 @@ _construct_pashua_id INVERT_PHASE       x=410 y=0 type=checkbox \
                                         default="${INVERT_PHASE}"
 _construct_pashua_id WARNING            x=350 y=0 type=text \
                                         default="WARNING: Do not use this option unless required"
+_construct_pashua_id LOG_CAPTURE_CONFIG_CHOICE     x=660 y=110 type=radiobutton \
+                                        label="Report capture configuration" \
+                                        tooltip="select whether capture configuration device information will be recorded in the logs. Requires configuration selection" \
+                                        default="${LOG_CAPTURE_CONFIG_CHOICE}" \
```

Comment:
If LOG_CAPTURE_CONFIG_CHOICE could you use CC_PROFILE_DIR_Default. Otherwise for the first use the openbrowser ui starts in a place unrelated to the profiles.